test/python/utils.py

```diff
@@ -1,4 +1,10 @@
 
+def app_path_from_app_name(app_dir, app_name: str, device: str):
```

Comment:
`def app_path_from_app_name(app_dir: Path, app_name: str, device: str) -> Path:`

Reply:
(I should have typed this code more carefully)